docs/market-search-analysis.md

```
@@ -0,0 +1,191 @@
+
+더미 데이터: 가게 10,000개, 상품 60,000개 (가게당 6개)
+
+## 베이스라인 측정
+
+### 전체 조회 (키워드 없음)
+
+```sql
+SELECT * FROM markets ORDER BY id LIMIT 10 OFFSET 0;
+```
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)